tutorials/eda_contents/Computing_Environment/Unix.md

```diff
@@ -0,0 +1,363 @@
+   ...
+~~~
+
+
+To navigate through the filesystem, there is the `cd` command which stands for 'change directory'. Note, this commmand
+doesn't actually change the directory just the shell's idea of what directory we are in.
+
+~~~
+$ cd ..
+jovyan@97d07e419e83:/notebook_dir/writable-workspace$
+~~~
+
+Remember the hidden directory `..` from the `ls -a` command? 
+`..` is a special directory name meaning "the directory containing this one", or more succinctly,
+the **parent** of the current directory. In Marble, this command should take you to `/notebook_dir/writable-workspace`
+
+~~~
+$ pwd
+/notebook_dir/writable-workspace
+~~~
+
+Indicating the parent directory in Marble is `/notebook_dir/writable-workspace`.
+
+Using the `cd` command without an argument returns you to the home directory.
+
+We can also specify the path to the directory wish, for example:
+
+~~~
+/notebook_dir/writable-workspace$ cd
+~$ pwd
+/notebook_dir/writable-workspace/.home
+~~~
+~~~
+~$ cd /notebook_dir/mypublic
+/notebook_dir/mypublic$ pwd
+/notebook_dir/mypublic
+~~~
+
+#### Key Points:
+- "The file system is responsible for managing information on the disk."
+- "Information is stored in files, which are stored in directories (folders)."
+- "Directories can also store other directories, which forms a directory tree."
+- "`cd path` changes the current working directory."
+- "`ls path` prints a listing of a specific file or directory; `ls` on its own lists the current working directory."
+- `pwd` prints the user's current working directory.
+- `whoami` shows the user's current identity.
+- `/` on its own is the root directory of the whole file system.
+- A relative path specifies a location starting from the current location.
+- An absolute path specifies a location from the root of the file system.
+- Directory names in a path are separated with '/' on Unix, but '\\\\' on Windows.
+- '..' means 'the directory above the current one'; '.' on its own means 'the current directory'.
+- Most files' names are `something.extension`. The extension isn't required, and doesn't guarantee anything, 
+but is normally used to indicate the  type of data in the file.
+- Most commands take options (flags) which begin with a '-'.
+
+## Working with Files and Directories
+
+You would have seen basic ways to navigate through files and directories in Unix. Now, let's make them.
+
+To create a directory, we use the command `mkdir` plus the name of the directory you wish to create.
+
+~~~
+/notebook_dir/mypublic$ mkdir example_of_directory
+/notebook_dir/mypublic$ ls
+'Environment'   example_of_directory   Marble.md   Plan.ipynb
+~~~
+
+Note the `mkdir` yielded no output but when we check for the list of files and directories within the current working directory, 
+we see `example_of_directory` was created. 
+Be aware that the `mkdir` command makes the new directory within your current working directory. Ensure you create the directory in 
+the correct location.
```

Comment:
Maybe add a small section on the `rmdir` command as well (so that users know how to remove a directory they may have created accidentally)

Reply:
added in updated version